toonz/sources/tnztools/vectortapetool.cpp

```diff
@@ -327,7 +327,7 @@ class VectorTapeTool final : public TTool {
     if (!vi) return;
 
     // BUTTA e rimetti (Dava problemi con la penna)
-    if (!m_draw) return;  // Questa riga potrebbe non essere messa
+    //if (!m_draw) return;  // Questa riga potrebbe non essere messa
```

Comment:
The function `VectorTapeTool::draw()` has the same condition and it occasionally block to draw the tool UI properly.
As far as I tested, I think we can just remove `VectorTapeTool::m_draw` and all lines related to it. I'll do so after merging this PR.
Thank you @flurick for the fix, LGTM